images/tier1/navigate-dark.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
Haha we really need dark and not dark versions of this? I thought all the tier1 icons were supposed to be dark.